packages/framer-motion/src/motion/utils/use-visual-element.ts

```diff
@@ -63,7 +63,7 @@ export function useVisualElement<Instance, RenderState>(
     useIsomorphicLayoutEffect(() => {
         if (!visualElement) return
 
-        microtask.postRender(visualElement.render)
+        microtask.render(visualElement.render)
```

Comment:
What is this change? (and the microtask.preRender change below)

Reply:
True, not sure these are needed any more. `render` is the "right" place for this one to fire but `preRender` I think was an artefact of when I was trying some things out.